Exercises/data-types-overview/data-types-overview.js

```diff
@@ -0,0 +1,170 @@
+
+
+// Declare a variable on each type using two options: plain creation, creation with constructor.
+
+//number
+let number = 123;
+console.log(number);
+
+//string
+let string = "some text";
+console.log(string);
+
+//boolean
+let bool = true;
+console.log(bool);
+
+//undefined
+let undef = undefined;
```

Comment:
better not to cut variables names, in your case I think better use `undefinedVariable`  and `nullVariable`